### ISSUE TITLE
[Tabs extended]: React wrapper VoiceOver - cannot tab to other tab options - DPO Consulting

### ISSUE THREAD

**Issue by RichKummer** [carbon-design-system/carbon-for-ibm-dotcom#8310]
**Engineering info:**  
- Innovation Team engineer: Pending Innovation team's 02/21 sprint planning
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2413
- DPO consulting engineer:

-----

### Description

When using `Tabs extended` in React wrapper on Firefox with VoiceOver enabled, I can only tab into the first tab content. Unclear how a keyboard user would navigate to the other tabs and access that content.

https://user-images.githubusercontent.com/15643582/154329471-767f2774-fc29-4d10-8892-010f27ef6d71.mov

### Component(s) impacted

Tabs extended

### Browser

Firefox

### Carbon for IBM.com version

v1.31.0-rc.1

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

C4IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://codesandbox.io/s/github/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/web-components/examples/codesandbox/components-react/tabs-extended

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by andy-blum** [comment]
@RichKummer thanks for opening this issue on suspected accessibility issues, but this is in fact expected behavior on a tabbed interface. The interaction model should be:

- tab key moves to `tablist` and `tabpanel`, but not each individual `tabitem`
- when `tablist` has focus, left/up arrow (or right/up in RTL) selects previous tab
- other arrow keys select next tab
- home/end keys jump to first/last tab respectively

Example tabbed interface from W3 available [here](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html)

Testing locally, I'm able to make this happen on the major browsers on desktop
